ZAM_fnc_EZMLiteV2.1.1.sqf

```diff
@@ -6013,6 +6014,267 @@ MAZ_EZM_fnc_initFunction = {
+
+		comment "show intertitles";
+		[_intertitles] spawn {
+			params ["_intertitles"];
+			private _line1 = [_intertitles # 0] call HYPER_fnc_splitMaxLine;
+			private _line2 = [_intertitles # 1] call HYPER_fnc_splitMaxLine;
+			sleep 3;
+			_line1 spawn BIS_fnc_infoText;
+			sleep 5;
+			_line2 spawn BIS_fnc_infoText;
+		};
+
+		comment "Post processing";
+		switch (_postProcess) do {
+			case "none": {
+				"colorCorrections" ppEffectAdjust[1,1,0,[0,0,0,0],[1,1,1,1],[0,0,0,0]];
+				"colorCorrections" ppEffectCommit 0;
+				"colorCorrections" ppEffectEnable true;
+			};
+			case "highcontrast": {
+				"colorCorrections" ppEffectAdjust [1, 0.9, -0.002, [0.0, 0.0, 0.0, 0.0], [1.0, 0.6, 0.4, 0.6],  [0.199, 0.587, 0.114, 0.0]];
+				"colorCorrections" ppEffectCommit 0;
+				"colorCorrections" ppEffectEnable true;
+			};
+			case "blue": {
+				"colorCorrections" ppEffectAdjust [1, 1, 0, [0.0, 0.0, 0.0, 0.0], [0.6, 0.6, 1.8, 0.7],  [0.199, 0.587, 0.114, 0.0]];
+				"colorCorrections" ppEffectCommit 0;
+				"colorCorrections" ppEffectEnable true;
+			};
+			case "dull": {
+				"colorCorrections" ppEffectAdjust [1, 0.8, -0.002, [0.0, 0.0, 0.0, 0.0], [0.6, 0.7, 0.8, 0.65],  [0.199, 0.587, 0.114, 0.0]];
+				"colorCorrections" ppEffectCommit 0;
+				"colorCorrections" ppEffectEnable true;
+			};
+			case "yellowgamma": {
+				"colorCorrections" ppEffectAdjust [1, 1, 0, [0.0, 0.0, 0.0, 0.0], [0.6, 1.4, 0.6, 0.7],  [0.199, 0.587, 0.114, 0.0]];
+				"colorCorrections" ppEffectCommit 0;
+				"colorCorrections" ppEffectEnable true;
+			};
+			case "greengamma": {
+				"colorCorrections" ppEffectAdjust [1, 1, 0, [0.0, 0.0, 0.0, 0.0], [1.8, 1.8, 0.3, 0.7],  [0.199, 0.587, 0.114, 0.0]];
+				"colorCorrections" ppEffectCommit 0;
+				"colorCorrections" ppEffectEnable true;
+
+			};
+		};
+
+		if (_cinematicType == "Flyby") then {
+			comment "in flyby mode, we select the module location as our target, and the camera paths are automatically designated at 0 and 90 degrees";
+			private _camTarget = "Land_HelipadEmpty_F" createVehicleLocal _target;
+			private _circleRadius = 200;
+			private _camHeight = 200;
+			private _camSrc0 = [_target select 0, (_target select 1) + _circleRadius, (_target select 2) + _camHeight];
+			private _camSrc90 = [(_target select 0) + _circleRadius, _target select 1, (_target select 2) + _camHeight];
+			
+			private _camera = "camera" camCreate _camSrc0;
+			_camera cameraEffect ["internal", "back"];
+			_camera camPrepareTarget _camTarget;
+			_camera camSetFov 1;
+			_camera camCommitPrepared 0;
+
+			_camera camPreparePos _camSrc90;
+			_camera camCommitPrepared 15;
+			waitUntil { camCommitted _camera };
+			cutRsc ["RscStatic", "PLAIN"];
+			sleep 0.4;
+			_camera cameraEffect ["terminate", "back"];
+			cutText ["", "BLACK IN", 2];
+			[1, 2, true, true] call BIS_fnc_cinemaBorder;
+
+		};
+
+
+
+		comment "clean up scripts";
+		if(_zeusCanSeeCutscene) then {
+			[1] call MAZ_EZM_fnc_setInterfaceToRefresh;	
```

Comment:
I haven't tested this yet, why use a 1 second delay to refresh the display instead of using the function `MAZ_EZM_fnc_refreshInterface` or just `(findDisplay 312) closeDisplay 0;`? Do those not work as good? 

You could potentially close the Zeus display on each `_zeusPlayer` from the comment above and then reopen it with `openCuratorInterface` after the cinematic completes.